source/clear-linux/tutorials/dlaas.rst

```diff
@@ -0,0 +1,240 @@
+TensorFlow\* and Kubeflow with the Intel® Deep Learning as a Service Stack.
+
+The Intel® Deep Learning as a Service Stack is available in two versions.
+The first is `Eigen`_, which includes `TensorFlow`_ optimized for Intel®
+architecture. The second is `Intel MKL-DNN`_, which includes the TensorFlow
+framework optimized using Intel® Math Kernel Library for Deep Neural
+Networks (Intel® MKL-DNN) primitives.
+
+.. contents:: :local:
+   :depth: 1
+
+Release notes
+=============
+
+View current `release notes`_ for the Intel® Deep Learning as a Service
+Stack.
+
+.. note::
+
+   Performance test numbers in the Intel® Deep Learning as a Service Stack were obtained using `runc` as the runtime.
+
+Prerequisites
+=============
+
+* |CL| installed on host system. If not installed, :ref:`bare-metal-install`
+* `containers-basic` bundle
+* `cloud-native-basic` bundle
+
+In |CL|, `containers-basic` provides Docker\*, which is required for
+TensorFlow benchmarking. Use the :command:`swupd` utility to check if
+`containers-basic` and `cloud-native-basic` are present:
+
+.. code-block:: bash
+
+   sudo swupd bundle-list
+
+If you need to install the `containers-basic` or `cloud-native-basic`, enter:
+
+.. code-block:: bash
+
+   sudo swupd bundle-add containers-basic cloud-native-basic
+
+To ensure that kubernetes is correctly installed and configured,
+:ref:`kubernetes`.
+
+We have validated these steps against the following software package
+versions:
+
+* |CL| 26240--lowest version permissible.
+* Docker 18.06.1
+* Kubernetes 1.11.3
+* Go 1.11.12
+
+TensorFlow single and multi-node benchmarks
+============================================
+
+This section describes running the `TensorFlow benchmarks`_ in single node.
+For multi-node testing, replicate these steps for each node. These steps
+provide a template to run other benchmarks, provided that they can invoke
+TensorFlow.
+
+#. Download and run either the `Eigen`_ or the `Intel MKL-DNN`_ docker image
+   from `Docker Hub`_.
+
+   .. note::
+
+      You will enter the following commands in the running container.
+
+      Replace <docker_name> with the name of the image.
+
+#. Clone the benchmark repository:
+
+   .. code-block:: bash
+
+      docker exec -t <docker_name> bash -c ‘git clone http://github.com/tensorflow/benchmarks -b cnn_tf_v1.11_compatible’
+
+#. Next, execute the benchmark script to run the benchmark.
+
+   .. code-block:: bash
+
+      docker exec -i <docker_name> bash -c ‘python benchmarks/scripts/tf_cnn_benchmarks/tf_cnn_benchmarks.py --device=cpu --model=resnet50 --data_format=NWHC ’.
+
+      .. note::
+
+         You can replace the model with one of your choice supported by the
+         TensorFlow benchmarks.
+
+Kubeflow multi-node benchmarks
+==============================
+
+The benchmark workload will run in a Kubernetes container. We will use
+`Kubeflow`_ and deploy three nodes for this tutorial to show resource
+management and get sufficient output data for evaluation.
```

Comment:
"The benchmark workload will run in a Kubernetes cluster. We will use Kubeflow for the Machine Learning workload deployment on three nodes..."

Reply:
Done.